File: app/src/main/java/com/example/moviles1/ui/Peliculas/Peliculas.java
package com.example.moviles1.ui.Peliculas;

import android.content.Context;
import android.net.Uri;
import android.os.Bundle;

import androidx.fragment.app.Fragment;
import androidx.viewpager.widget.ViewPager;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.moviles1.R;
import com.google.android.material.tabs.TabItem;
import com.google.android.material.tabs.TabLayout;


public class Peliculas extends Fragment {

TabLayout tabLayout;
ViewPager viewPager;
TabItem tab1,tab2;


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.peliculas_fragment, container, false);



    }

}

File: app/src/main/java/com/example/moviles1/ui/Peliculas/Peliculas.kt
package com.example.moviles1.ui.Peliculas

import androidx.lifecycle.ViewModelProviders
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.example.moviles1.R

class Peliculas : Fragment() {

    companion object {
        fun newInstance() = Peliculas()
    }

    private lateinit var viewModel: PeliculasViewModel

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.peliculas_fragment, container, false)
    }

    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        viewModel = ViewModelProviders.of(this).get(PeliculasViewModel::class.java)
        // TODO: Use the ViewModel
    }

}
